spec/requests/planning_application_list_spec.rb

```diff
@@ -32,68 +28,99 @@
 
   describe "data" do
     let(:data) { json["data"] }
+    let(:planning_application_json) { data.first }
 
-    it "returns an empty response if no planning application" do
-      get "/api/v1/planning_applications.json"
-
-      expect(response).to be_successful
-      expect(data).to be_empty
-    end
-
-    it "returns a list of serialized planning application" do
-      planning_application_1 = create(:planning_application, :determined)
-      planning_application_2 = create(:planning_application, :determined)
-
-      decision_granted = create(:decision, :granted, user: reviewer, planning_application: planning_application_1)
-      decision_refused = create(:decision, :refused, user: reviewer, planning_application: planning_application_2)
-
-      get "/api/v1/planning_applications.json"
-      expect(response).to be_successful
-
-      expect(data).to match(
-        a_collection_containing_exactly(
-          a_hash_including(
-            "id" => planning_application_1.id,
-            "application_number" => planning_application_1.reference,
-            "site_address" => planning_application_1.site.full_address,
-            "application_type" => "Certificate of Lawfulness",
-            "summary_of_proposal" => planning_application_1.description,
-            "received_date" => planning_application_1.created_at.getutc.iso8601,
-            "determined_at" => planning_application_1.determined_at.getutc.iso8601,
-            "status" => "granted"
-          ),
-          a_hash_including(
-            "id" => planning_application_2.id,
-            "application_number" => planning_application_2.reference,
-            "site_address" => planning_application_2.site.full_address,
-            "application_type" => "Certificate of Lawfulness",
-            "summary_of_proposal" => planning_application_2.description,
-            "received_date" => planning_application_2.created_at.getutc.iso8601,
-            "determined_at" => planning_application_1.determined_at.getutc.iso8601,
-            "status" => "refused"
-          )
-        )
-      )
+    def json_time_format(time)
+      time.utc.iso8601(3) if time.present?
     end
 
-    it "returns an empty response if the planning application is in assessment" do
-      planning_application_1 = create(:planning_application)
-      decision_granted = create(:decision, :granted, user: reviewer, planning_application: planning_application_1)
-
+    it "returns an empty response if no planning application" do
```

Comment:
I'm confused about this - should this be expected to be a 404?

Reply:
No, it would be a 404 if it was the show action, this is the index action